test/ui-test/testSuites/suite_onboarding/tst_statusSignUp/test.feature

```diff
@@ -122,8 +122,8 @@ Feature: Status Desktop Sign Up
     Given A first time user lands on the status desktop and navigates to import seed phrase
     When The user inputs the seed phrase "<seed>"
     And user clicks on the following ui-component seedPhraseView_Submit_Button
-    When user signs up with username "tester123" and password "TesTEr16843/!@00"
-    Then the user lands on the signed in app
+    Given the user signs up with username "tester123" and password "TesTEr16843/!@00"
```

Comment:
Not sure why this would be changed from a When / Then statement.

Suggestion:
```
Given A first time user lands on the status desktop and navigates to import seed phrase
And The user inputs the seed phrase "<seed>"
And user clicks on the following ui-component seedPhraseView_Submit_Button
When the user signs up with username "tester123" and password "TesTEr16843/!@00"
Then the user lands on the signed in app
```